lib/config.js

```diff
@@ -47,3 +47,9 @@ exports.getHomeDir = function(home) {
 exports.getNcuDir = function(dir) {
   return path.join(dir || process.cwd(), '.ncu');
 };
+
+exports.getGitConfig = function(dir) {
+  const home = exports.getHomeDir(dir);
+  const configPath = path.join(home, '.gitconfig');
```

Comment:
Hmm..what happens if the user uses different sign keys in different directories?